eZ/Publish/API/Repository/Tests/Parallel/ParallelProcessList.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * File containing the ParallelProcessList class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests\Parallel;
+
+use Jenner\SimpleFork\Process;
+
+class ParallelProcessList implements \IteratorAggregate
+{
+    /** @var Jenner\SimpleFork\Process[] */
+    private $pool = [];
+
+    public function addProccess(Process $callback): void
+    {
+        $this->pool[] = $callback;
+    }
+
+    public function getIterator()
```

Comment:
```suggestion
    public function getIterator(): \Iterator
```
or something more relevant e.g `Traversable`?